internal/view/live_view_test.go

```diff
@@ -14,14 +14,11 @@ func TestLiveView_linesWithRegions(t *testing.T) {
 		e       []string
 	}{
 		"empty-lines": {
-			lines:   []string{},
-			matches: fuzzy.Matches{},
-			e:       []string{},
+			e: []string{},
```

Comment:
Same here. This is the zero value.

Reply:
Same as above, the zero value here is a `nil` slice.